pep-8015.rst

```diff
@@ -316,6 +316,9 @@ As any other member of the Python community, the PSF Code of Conduct
 Workgroup can ban a core developer for a limited amount of time. In this
 case, the core developer immediately looses their core developer status.
 
+A ban is only the last resort action when all other options have been
```

Comment:
Perhaps only:  In general, a ban is a last resort action.

There can be cases of egregious behavior where a ban could be the first action i.e. physical altercations.